jenkins/jenkins_build.sh

```diff
@@ -10,8 +10,8 @@ if [ "$skip_docker_build" = false ]; then
        broadinstitute/dsde-toolbox:latest vault read --format=json ${GCR_SVCACCT_VAULT} \
        | jq .data > dspci-wb-gcr-service-account.json
 
-  ./docker/build.sh jar -d push -n "us.gcr.io/broad-dsp-gcr-public" \
-      -k "dspci-wb-gcr-service-account.json"
+  ./docker/build.sh jar -d push -dr "broadinstitute" -gr "gcr.io/broad-dsp-gcr-public" \
```

Comment:
Is this pushing to gcr twice? with `-gr` and with `-n`

Reply:
nope-- the `-n` flag is for the repo to push notebooks to

Reply:
The notebooks image is using `us.gcr.io`, while the leo one is using `gcr.io` (global load balancer). Is 1 preferred over the other? I remember not being sure when I did the notebooks repo in GCR.

Reply:
i think the default when you push to GCR us `gcr.io`, which is what i went with for all the service image pushes